src/DropDown/DropDown.component.tsx

```diff
@@ -24,7 +24,7 @@ interface DropDownProps extends React.HTMLAttributes<HTMLDivElement> {
     border?: string;
     borderRadius?: string;
     debug?: boolean;
-    onTriggered?: (state: boolean) => void;
+    onTriggered?: (state: boolean, trigger?: string) => void;
```

Comment:
This is something we can unit test, and should. What's the end-goal of changing this util method?

Reply:
We have a button in a navbar that you click/hover to trigger the dropdown, and we want the button to respond to the dropdown (color change, chevron rotate, etc). Wanting to use `onTriggered` to track the state of the dropdown, but it is not useful when using the 'both' value for the `trigger` prop.

Happy to write unit tests if you're ok with the change

Reply:
If the button knows which trigger type is firing `onTriggered`, we can write a bit of logic to respond appropriately